osu.Game/Online/API/APIAccess.cs

```diff
@@ -453,7 +453,7 @@ public void Logout()
             // Scheduled prior to state change such that the state changed event is invoked with the correct user and their friends present
             Schedule(() =>
             {
-                localUser.Value = createGuestUser();
+                setLocalUser(createGuestUser());
```

Comment:
Schedule in a schedule is kinda weird...

Reply:
Yeah, why i added the `,false` to the nested one so it can run inline.

I can inline each of the calls if that sits better.